ui/src/app/auth/auth-roles.ts

```diff
@@ -3,3 +3,13 @@ export const OSMT_ADMIN = "ROLE_Osmt_Admin"
 export const OSMT_CURATOR = "ROLE_Osmt_Curator"
 
 export const ENABLE_ROLES = false
+
+export const ROLES_AUTHORITY : any  = {
```

Comment:
I would have called this by the mapping that it is:  ACTION_ROLES, or as you mention below, an ACTION_AUTHORITY.  That way the key is the action and the value is the array of roles representing the authority, similar to the term "role-based authorization", which is what this is all about.

Reply:
We refactored it to ACTION_ROLES